src/components/ClientCard/styles.js

```diff
@@ -12,7 +12,7 @@ const style = (Component) => styled(Component)`
   }
 
   h4 {
-    font-size: 18px;
+    font-size: 18px!important;
```

Comment:
We should avoid using the ```!important``` statement, as it can harm code maintainability, because it gets more difficult to know which styles are being applied. Instead the styles should be refactored to despise the need of them.

Reply:
Fixed at 3e49011